homework/index.js

```diff
@@ -30,21 +30,43 @@
   }
 
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const liRepository = createAndAppend('li', ul);
+    createAndAppend('b', liRepository, { text: ' Repository:  ' });
```

Comment:
Do not use the `<b>` tag to simply make your text bold. Follow the advice given here:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b